test/Interop/Cxx/foreign-reference/frt-retained-unretained-attributes-error.swift

```diff
@@ -2,42 +2,54 @@
 // RUN: not %target-swift-frontend -typecheck -I %S/Inputs  %s -cxx-interoperability-mode=upcoming-swift 2>&1 | %FileCheck %s
 
 import FunctionsAndMethodsReturningFRT
+import CxxStdlib
```

Comment:
Is this import necessary with this change?

Reply:
Yes, without it I was seeing a crash when initializing `std::function` `FunctionVoidToFRTStruct` from Swift
Ref:  rdar://140452227

Reply:
Ah I see, thanks for explaining.